packages/hoppscotch-common/src/components/environments/teams/index.vue

```diff
@@ -43,15 +43,17 @@
         />
       </div>
     </div>
-    <div
+    <HoppSmartPlaceholder
       v-if="!loading && !teamEnvironments.length && !adapterError"
-      class="p-5 space-y-25 flex flex-col"
+      :src="`/images/states/${colorMode.value}/blockchain.svg`"
+      :alt="`${t('empty.environments')}`"
+      :text="t('empty.environments')"
     >
       <div class="flex flex-col items-center space-y-4">
         <span class="text-secondaryLight text-center">
           {{ t("environment.import_or_create") }}
         </span>
-        <div class="flex gap-4 flex-col items-center">
+        <div class="flex gap-4 flex-col items-stretch">
```

Comment:
Is the `items-stretch` class necessary here?

Reply:
Yep, making the button to stretch the parent width is intentional.